src/components/WithdrawMilestoneFundsButton.jsx

```diff
@@ -38,15 +41,10 @@ const WithdrawMilestoneFundsButton = ({ milestone, isAmountEnoughForWithdraw })
       ])
         .then(([, donationsCount]) => {
           if (!isAmountEnoughForWithdraw) {
-            message.error(
-              `Oh No!
-                A minimum donation balance of
-                ${minimumPayoutUsdValue} USD  is required before
-                you can collect or disperse the funds. This is a
-                 temporary limitation due to Ethereum Mainnet issues.`,
-            );
+            minPayoutWarningInWithdraw(minimumPayoutUsdValue);
```

Comment:
`minimumPayoutUsdValue` can be obtained from white list provider inside notification provider.